src/application/WebMenuShortcuts.tsx

```diff
@@ -0,0 +1,57 @@
+import { useHotkeys } from 'react-hotkeys-hook';
+import type { OptionsOrDependencyArray } from 'react-hotkeys-hook/dist/types';
+
+import { emitEvent, RefStudioEventName, RefStudioEventPayload } from '../events';
+
+/**
+ * Add keyboard shortcuts to the app.
+ *
+ *  NOTE: Some shortcuts can+t be added to the app
+ *    - ⌘ + N - replaced by Ctrl + N
+ *    - ⌘ + W - replaced by Ctrl + N
+ *
+ *  WARNING:  https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/basic-usage
+ *  > Please be aware that there are some hotkeys that we cannot override,
+ *    because they would interfere with a safe browsing experience for the user.
+ *    These depend on the browser. For example in Chrome, most notably those are
+ *    meta + w which closes a tab, meta + n for opening a new window and
+ *    meta + t to open a new tab. Additionally meta + shift + w (closing all tabs of the current window),
+ *    meta + shift + n (opening incognito window) and meta + shift + t (reopen the last closed tab)
+ *    cannot be overridden.
+ *    meta + up + 1..9 on the other hand focuses the corresponding tab of the active window and
+ *    also cannot be overridden.
+ */
+export function WebMenuShortcuts() {
+  // RefStudio
+  useMenuEventHotKeyForWeb(['meta+,'], 'refstudio://menu/settings');
+  // File
+  useMenuEventHotKeyForWeb(['ctrl+n'], 'refstudio://menu/file/new');
+  useMenuEventHotKeyForWeb(['ctrl+s', 'meta+s'], 'refstudio://menu/file/save');
+  useMenuEventHotKeyForWeb(['ctrl+w'], 'refstudio://menu/file/close');
```

Comment:
I think these wouldn't work on Windows, since Windows use ctrl for this kind of shortcuts

Reply:
That's a good point. I feel that, for the web, we might drop (not use) most of these shortcuts for window/file management.